frontend/src/main/components/Levels/SingleLevelDropdown.js

```diff
@@ -0,0 +1,45 @@
+import React, {useState} from "react";
+import {Form} from "react-bootstrap";
+
+const SingleLevelDropdown = ({levels, _level, setLevel, controlId, onChange = null, label="Course Level"}) => {
```

Comment:
Is this supposed to be `_level` instead of `level`?

Reply:
Yes, fixed.